ui/types.go

```diff
@@ -0,0 +1,14 @@
+package ui
+
+// TextOpts Asking for text options
+type TextOpts struct {
+	Label   string
+	Default string
+}
+
+// ChoiceOpts asking for choice options
+type ChoiceOpts struct {
+	Label   string
+	Default string
```

Comment:
We did think about it. But went ahead with string because from an end-user perspective, when he gives choices, it will be easy for him to give the default as `string` rather than `int`. If we choose int, then he has to relook at the choices and count the index where his default will fall. 

But yes, since the return type is `int`, we can go ahead with it.